py/setup.cfg

```diff
@@ -1,5 +1,5 @@
 [flake8]
-exclude = .tox,docs/source/conf.py
+exclude = .tox,docs/source/conf.py,*venv
```

Comment:
This didn't actually work for me. I think this did: `./venv`

Reply:
I can't recreate this; where does your virtual env typically reside, can you share the full path? I tested it yesterday and couldn't break it